### DISCUSSION TITLE
Reorder stops in route

### DISCUSSION THREAD

**Discussion by trymeouteh** [organicmaps/organicmaps#8674] [Ideas]
When adding stops in a route, it is quite confusing and annoying to get the stops in order.

I would like to suggest a menu at the bottom to reorder stops in a route

**Comment by biodranik** [comment]
@trymeouteh what is your typical scenario of adding stops? How do you do it, and why is it confusing?

Most apps (including OM) do it like this:
1. You build a route (either from a current position or from some starting point, the latter is easier in OM than in other apps).
2. You add stops inside the already-built route (first stop, then second stop, then third).

OM on iOS does exactly like this with a way to reorder stops.

OM on Android automatically optimizes the order of stops and doesn't have a way to reorder them yet.

- [ ] #2551 
- [ ] https://github.com/organicmaps/organicmaps/issues/7235
- [ ] https://github.com/organicmaps/organicmaps/issues/6349
